Cypress POC (#1937)
kyma-project/console: tests/README.md

## Development

To run UI tests inside a container in the cluster using a Chrome browser, use this command:

``` bash
npm run test:cluster
```

> To run UI tests during development against a cluster, use this command:

Reviewer: ```suggestion
To run UI tests on a cluster, use this command:
```

Reviewer: I made it simple. The section is already called **Development** so I guess we don't have to repeat it here.

``` bash

> CYPRESS_DOMAIN=<your domain> CYPRESS_PASSWORD=<your cluster password> npm start

Reviewer: ```suggestion
CYPRESS_DOMAIN={YOUR_DOMAIN} CYPRESS_PASSWORD={YOUR_CLUSTER_PASSWORD} npm start
```
